src/thor/triplegbuilder.cc

```diff
@@ -659,10 +659,15 @@ void AddTripIntersectingEdge(const AttributesController& controller,
   // Set the sign info for the intersecting edge if requested
   if (controller.attributes.at(kNodeIntersectingEdgeSignInfo)) {
     if (intersecting_de->sign()) {
-      GraphId beginnode = graphreader.GetBeginNodeId(intersecting_de, graphtile);
-      valhalla::baldr::graph_tile_ptr t2 = graphreader.GetGraphTile(beginnode);
-      size_t edge_idx = intersecting_de - t2->directededge(0);
-      std::vector<SignInfo> edge_signs = t2->GetSigns(edge_idx);
+      // To retrieve the sign information from the intersecting_de I need
+      // to access the graph-tile for the "begin node" of the intersecting_de.
+      // GraphReader::GetGraphTile(DirectedEdge *) returns the graph-tile for
+      // the "end node". Hence, I need to obtain the opposing-directed-edge
+      // and then ask for it's graph tile. (Sigh)
+      GraphId opp_intersecting_de = graphtile->GetOpposingEdgeId(intersecting_de);
+      graph_tile_ptr opp_tile = graphreader.GetGraphTile(opp_intersecting_de);
+      size_t edge_idx = intersecting_de - opp_tile->directededge(0);
```

Comment:
here we have another problem. if `intersecting_de` is not in the same tile as its opposing edge then this pointer arithmetic would be completely off because `intersecting_de` doesnt actually occur inside of `opp_tile`s directected edges array